modules/custom-post-types/portfolios.php

```diff
@@ -713,6 +723,19 @@ static function get_project_tags( $post_id ) {
 	}
 
 	/**
+	 * Displays the author of the current portfolio project.
+	 *
+	 * @return html
+	 */
+	static function get_project_author() {
+		$html = '<div class="project-author"><span>' . esc_html__( 'Author:', 'jetpack' ) . '</span> ';
```

Comment:
@eliorivero yes, that would work, but you need to add a comment for translators exactly above the call to the translation function like this: https://github.com/Automattic/jetpack/blob/95a9758d97775cbf5d83b1eb99de5d968de01436/modules/custom-post-types/comics.php#L307